### PR TITLE
Converts code imports for testings

### PR DESCRIPTION
**Pull request by aureliosaraiva**
For: https://github.com/emberjs/guides/pull/2004

### CODE REVIEW

File: source/testing/testing-controllers.md [hunk 1]
```diff
@@ -119,10 +119,10 @@ test('should modify the post model', function(assert) {
   const postCtrl = ctrl.get('post');
 
   // wrap the test in the run loop because we are dealing with async functions
-  Ember.run(function() {
+  run(function() {
 
     // set a generic model on the post controller
-    postCtrl.set('model', Ember.Object.create({ title: 'foo' }));
+    postCtrl.set('model', EmberObject.create({ title: 'foo' }));
```
Comment (acorncom) [inline source/testing/testing-controllers.md]: Thanks @aureliosaraiva, could you add the imports to the appropriate tests as well?  You've got it in your initial change, but subsequent files will need it as well ...

Comment (aureliosaraiva) [inline source/testing/testing-controllers.md]: I made an adjustment, let me know if I understood correctly.

File: source/testing/testing-components.md [hunk 2]
```diff
@@ -127,7 +127,7 @@ test('should update title on button click', function(assert) {
   assert.equal(this.$('h2').text(), 'Hello World', 'initial text is hello world');
 
   //Click on the button
-  Ember.run(() => document.querySelector('.title-button').click());
+  run(() => document.querySelector('.title-button').click());
```
Comment (acorncom) [inline source/testing/testing-components.md]: @aureliosaraiva Looks like you aren't doing the import for this test as well?

File: source/testing/testing-components.md [hunk 3]
```diff
@@ -187,7 +187,7 @@ test('should trigger external action on form submit', function(assert) {
   this.$('textarea').change();
 
   // click the button to submit the form
-   Ember.run(() => document.querySelector('.comment-input').click());
+   run(() => document.querySelector('.comment-input').click());
```
Comment (acorncom) [inline source/testing/testing-components.md]: Or here in this test file

File: source/testing/testing-controllers.md (hunk 1 above)
Comment (acorncom) [inline source/testing/testing-controllers.md]: Yep, looks good here

File: source/testing/testing-components.md [hunk 4]
```diff
@@ -276,7 +338,44 @@ test('should reveal current location', function(assert) {
 In the next example, we'll add another test that validates that the display changes
 when we modify the values on the service.
 
-```tests/integration/components/location-indicator-test.js
+```tests/integration/components/location-indicator-test.js{+38,+39,+40,+41,+42,+43,+44,+45}
```
Comment (aureliosaraiva) [inline source/testing/testing-components.md]: @acorncom I thought I needed the diff in this example.

**Comment by aureliosaraiva** [comment]
Closed in favor: https://github.com/emberjs/guides/pull/2042

**Pull request closed without merging**